trying to run on non network computer

Hi I am trying to run fgbio in non network computer and I am getting following error. I think it is looking for local host and I think it is coming from your cmdline fgbiomain class when looking for DMS. I tried to built the jar by commenting out this part of the code but I am failed to build jar using sbt. 
Exception in thread "main" java.net.UnknownHostException: removed url address: removed url address : unknown error
        at java.net.InetAddress.getLocalHost(InetAddress.java:1505)
        at com.fulcrumgenomics.cmdline.FgBioMain.printStartupLines(FgBioMain.scala:175)
        at com.fulcrumgenomics.cmdline.FgBioMain.makeItSo(FgBioMain.scala:142)
        at com.fulcrumgenomics.cmdline.FgBioMain.makeItSoAndExit(FgBioMain.scala:119)
        at com.fulcrumgenomics.cmdline.FgBioMain$.main(FgBioMain.scala:83)
        at com.fulcrumgenomics.cmdline.FgBioMain.main(FgBioMain.scala)
Caused by: java.net.UnknownHostException: removed url address: unknown error
        at java.net.Inet4AddressImpl.lookupAllHostAddr(Native Method)
        at java.net.InetAddress$2.lookupAllHostAddr(InetAddress.java:928)
        at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1323)
        at java.net.InetAddress.getLocalHost(InetAddress.java:1500

Is there any way you can provide the jar without host lookup so fgbio can be run on non network computer as due to security we cannot run on network computer.

--- a/src/main/scala/com/fulcrumgenomics/cmdline/FgBioMain.scala
+++ b/src/main/scala/com/fulcrumgenomics/cmdline/FgBioMain.scala
@@ -177,7 +177,7 @@ class FgBioMain extends LazyLogging {
   /** Prints a line of useful information when a tool starts executing. */
   protected def printStartupLines(tool: String, args: Array[String], commonArgs: FgBioCommonArgs): Unit = {
     val version    = CommandLineProgramParserStrings.version(getClass, color=false).replace("Version: ", "")
-    val host       = InetAddress.getLocalHost.getHostName
+    val host       = try { InetAddress.getLocalHost.getHostName } catch { case _: Exception => "unknown-host" }
     val user       = System.getProperty("user.name")
     val jreVersion = System.getProperty("java.runtime.version")
     val snappy     = if (new SnappyLoader().isSnappyAvailable) "with snappy" else "without snappy"

